### PR TITLE
Bugifx: Check the version correctly and not with a simple compare.

### PR DESCRIPTION
**Pull request by SaschaMzH**

### CODE REVIEW

**Comment by SaschaMzH** [comment]
[Version check](https://trello.com/c/xSs1K13N/21-version-check)

**Comment by JonasTrampe** [comment]
What about: A="$(echo 'one_two_three_four_five' | cut -d'_' -f0)"

**Comment by JonasTrampe** [comment]
What about this pullrequest?

**Comment by SaschaMzH** [comment]
@JonasTrampe 
Now it is also working on the Omega.

File: update.sh [hunk 1]
```diff
@@ -24,9 +26,29 @@ while :; do
...
 done
 
-# Get the latest/current version from the website.
-latestVersion=$(curl -k https://github.com/$UPDATE_SOURCE_REPO/releases/latest 2>/dev/null | sed "s/[a-zA-Z<> \"\/=:]//g" | sed "s/^\.//g" | sed "s/\.$//g")
-currentVersion="unknown"
+# get the list of all available versions and sort the list
+versions=$(curl --silent "https://api.github.com/repos/$UPDATE_SOURCE_REPO/releases" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' | sort -r)
+
+# determine the latest version based on the settings from the command line
+for version in $versions
```
Comment (JonasTrampe) [inline update.sh]: I do understand what this loop does, but I needed to read it twice, comments would be recommended

**Comment by SaschaMzH** [comment]
Now with comments.

File: update.sh (hunk 1 above)
Comment (SaschaMzH) [reply update.sh]: Please see the latest commit.

**Pull request merged**